### DISCUSSION TITLE
docker中Config.toml的路径是什么？还是需要自己添加这个文件？？？

### DISCUSSION THREAD

**Discussion by wwkk2580** [go-olive/olive#97] [Q&A]
docker中Config.toml的路径是什么？还是需要自己添加这个文件？？？

**Comment by wwkk2580** [comment]
运行docker-compose up -d olive-portal
报错：Error invoking remote method 'docker-start-container': Error: (HTTP code 400) unexpected - failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error mounting "/run/desktop/mnt/host/d/docker/olive/config/default.conf" to rootfs at "/etc/nginx/conf.d/default.conf": mount /run/desktop/mnt/host/d/docker/olive/config/default.conf:/etc/nginx/conf.d/default.conf (via /proc/self/fd/14), flags: 0x5000: not a directory: unknown: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type

**Comment by luxcgo** [comment]
报错的是portal的配置文件错误，对应的这个文件 https://github.com/go-olive/olive/blob/main/zarf/docker/config/default.conf
如果不需要修改的话把以下内容删掉即可
```yaml
volumes:
      - ./config/default.conf:/etc/nginx/conf.d/default.conf
```
和config.yaml没关系。
